dev/config/MASTER.yaml

```diff
@@ -1,15 +1,50 @@
----
-
 ssh_options:
    agent: False
 
 nodes:
   - hostname: localhost
 
+mesos_options:
+  master_address: mesos.paasta-mesosstage.yelp
+  master_port: 5050
+  secret: ''
+  role: '*'
+  enabled: True
+  default_volumes:
```

Comment:
Let's take out the default volumes unless we need them, I just added these a while back when I was testing the default volumes feature. (Same with the mrjob one below in the action.) It could cause errors if those paths don't exist on the agent.